app/assets/javascripts/calendar.es6

```diff
@@ -14,14 +17,44 @@ class Calendar {
       height: 'auto',
       allDaySlot: false,
       firstDay: 1,
-      defaultDate: moment(el.data('default-date'))
+      defaultDate: moment(el.data('default-date')),
+      viewRender: (view) => {
```

Comment:
The issue with having the full method in the default config is that this'll get blown away if a `viewRender` is passed in with the config.

Might be worth splitting this out, calling the 'default' one, and then running the passed in `config.viewRender` after the fact (if defined).